Monthly Projections.Rmd

```diff
@@ -385,18 +450,19 @@ sum(proj.env$MonthlyProjections$IUEPR_Month)
 
 This step projects share of monthly written premium and revises projections with reported data.
 
-The projection evenly distributes the estimated total written premium across the 12 month treaty period.
+The projection evenly distributes the estimated total written premium across the n month treaty period.
 
 ```{r, Project Business Proportion of Written Premium}
 # Define a function used for projecting monthly written premium
 # Calculates the expected fraction of written premium in a given month by checking if that month
-# is between the effective date and the expiration date (assuming a 12 month Treaty period)
+# is between the effective date and the expiration date
 # EFF = The effective date of the policy, assumed to be the first of the month
 # CAL = The calendar month for which we are calculating the expected written premium, assumed to be the first of the month
-WrittenPattern <- function(EFF, CAL) {
+WrittenPattern <- function(EFF, CAL, TLEN) {
+  TLEN <- min(TLEN, 12)
```

Comment:
@Nihilist9999 thinks we should take this out, and let it be any number of months even greater than 12